app/models/wiki/file.js

```diff
@@ -24,7 +25,7 @@ export default BaseModel.extend({
 			pageProperties = {
 				articleType: 'file',
 				fileUsageList: get(data, 'nsSpecificContent.fileUsageList')
-					.map(this.prepareFileUsageItem),
+					.map((item) => this.prepareFileUsageItem),
```

Comment:
What is this change for? `item` isn't being used.

Reply:
So, that `prepareFileUsageItem` has the right context, otherwise `this` is undefined in `prepareFileUsageItem`: https://stackoverflow.com/a/30148997